setup.py

```diff
@@ -39,11 +39,11 @@ def _dbt_duckdb_version():
     install_requires=[
         "dbt-common>=1,<2",
         "dbt-adapters>=1,<2",
-        "duckdb>=0.7.0",
+        "duckdb!=0.10.3",
```

Comment:
I thought it was like `duckdb>=0.7.0,!=0.10.3`?

Reply:
Yeah you're right, I forgot that `duckdb<0.7.0` is no bueno